### PR TITLE
add new company - Ocient

### PR DESCRIPTION
**Pull request by pkhanpara-ocient**
This Pull Request should adhere to the repository's [Code of Conduct](../../main/.github/CONTRIBUTING.md).

Title your pull request descriptively. `[edit X company]` or `[add new company - COMPANY]` 

Please read and check the boxes in the list below that apply!

If updating or adding a company, please include details here: Added company Ocient. 

Check off this list as appropriate (add an `x` between the `[]`). It will be used by @dougaitken when reviewing your contribution.

- [] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply.**
- [x] You are an employee of the company mentioned and confirm all included details are correct.
- [x] You know your alphabet - company is listed in alphabetical order in the README.md file.
- [x] The company directly hires employees. ** No bootcamps / freelance sites / etc**
- []x The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
- [x] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
- [x] An entry has been added or edited within the [readme.md file](../../main/README.md).
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

This is a modified version of the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -445,6 +445,7 @@ Name | Website | Region
 [Nuna](/company-profiles/nuna.md) | https://www.nuna.com/ | USA
 [O'Reilly Media](/company-profiles/oreilly-media.md) | https://www.oreilly.com/ | USA, UK, JPN, CHN
 [O'Reilly Online Learning](/company-profiles/oreilly-online-learning.md) | https://www.oreilly.com/ | Worldwide
+[Ocient] (/company-profiles/ocient.md) | https://ocient.com | USA, Europe, India
```
Comment (KieranRobson) [inline README.md:448]: Theres a typo `[Ocient] (/company-profiles/ocient.md)`

Comment (KieranRobson) [inline README.md:448]: `Oncient` Should be above the `O'reilly` entries

**Review by KieranRobson — COMMENTED** [review]
Tests are failing due to the following reasons:

**Comment by pkhanpara-ocient** [comment]
Thanks, @KieranRobson! I have addressed both issues with the latest commit.

File: README.md [hunk 2]
```diff
@@ -443,6 +443,7 @@ Name | Website | Region
 [npm](/company-profiles/npm.md) | https://www.npmjs.com/ | USA
 [Nuage](/company-profiles/nuage.md) | https://nuagebiz.tech/ | India
 [Nuna](/company-profiles/nuna.md) | https://www.nuna.com/ | USA
+[Ocient](/company-profiles/ocient.md) | https://ocient.com | USA, Europe, India
```
Comment (KieranRobson) [inline README.md:446]: `Company is listed out of order: "O'Reilly Media" (should be before "Ocient")` The logs are now saying that Oncient should be after `Oncient` although earlier it said they had to be after 😆

**Comment by pkhanpara-ocient** [comment]
With [7027ce8](https://github.com/remoteintech/remote-jobs/pull/1521/commits/7027ce8c1d847779c6ea877df6bd4252073138d8) commit O'Reilly entries will be before Ocient. I hope the tests pass now.

**Review by dougaitken — APPROVED** [review]

File: company-profiles/ocient.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## Region
+
+* North America
+* Europe
+* India 
+
+## Company technologies
+
+C++, Python, SQL
```
Comment (cj81499) [inline company-profiles/ocient.md:23]: Worth adding Bazel and Java?

Comment (pkhanpara-ocient) [reply company-profiles/ocient.md]: I have added java and bazel. I hope that's okay @KieranRobson

**Pull request merged**